src/main.ts

```diff
@@ -124,6 +124,11 @@ export async function run(): Promise<void> {
 
     const newBatchID = newBatch.batchID
 
+    core.info(`Launched batch ${newBatchID}`)
+    core.info(
+      `View results on ReSim: https://app.resim.ai/results/${newBatchID}`
```

Comment:
i think this'll be incorrect if the action is being used in a different deployment?